framework/src/media/codecs/opus_decoder_api.c

```diff
@@ -0,0 +1,241 @@
+/******************************************************************
```

Comment:
As I mentioned in report, a large stack size is necessary for opus encoding/decoding. 
Currently, I'm using a seperated thread (pthread) with 48k stack for encoding/decoding.
Otherwise, we need to enlarge the playerworker thread (std::thread) stack size.